src/luascript.cpp

```diff
@@ -13777,66 +13812,952 @@ int LuaScriptInterface::luaPartySetSharedExperience(lua_State* L)
 // Spells
 int LuaScriptInterface::luaSpellCreate(lua_State* L)
 {
-	// Spell(words, name or id)
-	InstantSpell* spell = nullptr;
+	// Spell(words, name or id) to get an existing spell
+	// Spell(type) ex: Spell(SPELL_INSTANT) or Spell(SPELL_RUNE) to create a new spell
```

Comment:
Wouldn't it be better if there was a separate function that returns existing spells? This way it's explicit in user code.

Reply:
That would break backward compatibility as `Spell` was already existing and was a returning function for instant Spells actually.
I think we should start to unify functions from now on, so we are not relying on something like `Item`, `ItemType` and keep it as `Item` so we rather have it returning whatever we push inside already.